controllers/recipeController.js

```diff
@@ -9,57 +9,64 @@ const Step = require('../models/stepModel.js');
 const Like = require('../models/likeModel.js');
 const Picture = require('../models/pictureModel.js');
 
+const numberOfRecipe = 10;
+let page = 0;
+
 /**
     Gets the list of recipes
 */
-const recipe_list_get = async (req, res) => {
+const recipe_list_get = async(req, res) => {
     /* #swagger.parameters['recipe'] = { 
            in: 'body',
            description: 'an array of recipes',
            type: 'object',
            schema: [{$ref: "#/definitions/recipe"}]
     } */
-  // TODO: add pagination
-  const recipes = await Recipe.scope('includeForeignKeys').findAll();
-  return res.json(recipes);
+    // TODO: add pagination
+
+    const recipes = await (await (await Recipe.scope('includeForeignKeys').findAll()).slice(page * numberOfRecipe, (page + 1) * numberOfRecipe));
+    page = page + 1;
+    if (recipes.length === 0) page = 0;
+    return res.json(recipes);
+
 };
 
-const recipe_get = async (req, res) => {
+const recipe_get = async(req, res) => {
     /* #swagger.parameters['recipe'] = { 
-           in: 'body',
-           description: 'Recipe',
-           type: 'object',
-           schema: { $ref: "#/definitions/recipe" }
+                   in: 'body',
+                   description: 'Recipe',
+                   type: 'object',
+                   schema: { $ref: "#/definitions/recipe" }
     } */
 
-  const recipeId = req.params.recipeId;
-  const recipe = await Recipe.scope('includeForeignKeys').findOne({
-    where: { id: recipeId },
-  });
-  console.log('recipeId', recipeId);
+    const recipeId = req.params.recipeId;
```

Comment:
Also it seems you've ran some sort of an auto-formatting script on the file. This makes the diff very hard to read as it has changed almost every line. Try not to do that in the future.